lib/src/rate_limited_scheduler.dart

```diff
@@ -59,7 +60,7 @@ class RateLimitedScheduler<J, V> {
   final Set<J> _started = {};
 
   RateLimitedScheduler(Future<V> Function(J) runJob,
-      {maxConcurrentOperations = 10})
+      {@required int maxConcurrentOperations})
```

Comment:
hehe, I guess @sigurdm opted to remove the default value in the function definition, instead of a removing the value at the callsite.. so merging the two branches I had to pick one :D

I'm not sure it matters a lot. But I suppose it's nice to explicit in this case.